common/templates/context.go

```diff
@@ -657,6 +657,11 @@ func baseContextFuncs(c *Context) {
 	c.addContextFunc("getPinCount", c.tmplGetChannelPins(true))
 	c.addContextFunc("getRole", c.tmplGetRole)
 	c.addContextFunc("getThread", c.tmplGetThread)
+	c.addContextFunc("createThread", c.tmplCreateThread)
+	c.addContextFunc("createMessageThread", c.tmplCreateMessageThread)
```

Comment:
maybe createThread can take a msgID and if it is nil, it will create a non-message thread, the way they both work otherwise is same.

Reply:
Speaking on forumTags. I made a PR that ended up being closed due to activity a while bit ago. The PRs aim was to implement setting tags via the same premise that `setRoles` did as well as retrieve the current forums tags. It was albeit not a perfect way of going about it due to my little experience with Go but if you do need a starting point it's there

Reply:
Thanks I'll find that PR and look through it. Maybe it'll give me some ideas

Reply:
Here: https://github.com/botlabs-gg/yagpdb/pull/1453

Reply:
Pushed changes that bring in some of the ideas/feedback from #1453 . `complexThread` now supports a tags input. When a forum thread with tags is created, the returned templates.CtxChannel.AppliedTags is populated with a list of discordgo.ForumTag.